pandas/tests/scalar/timestamp/test_comparisons.py

```diff
@@ -12,6 +12,12 @@
 from pandas import Timestamp
 
 
+utc_objs = ['utc', utc, tzutc()]
```

Comment:
can you make this a fixture and put it in pandas/conftest.py. thought we had this already.

Reply:
sure.

You might be thinking of the datetime_tz_utc fixture which just provides the datetime.timezone.utc object (which seems like a really unnecessary fixture to me, especially seeing as how it is only used once)